plugins/postcss-gap-properties/src/index.js

```diff
@@ -17,6 +17,10 @@ function creator(opts) {
 			}
 
 			const isNotDisplayGrid = !(decl.parent.some((node) => {
+				if (node.type !== 'decl') {
```

Comment:
When this is a `rule` or `at-rule` `prop` and `value` won't exist

Reply:
With this change I'm getting a runtime error on line 32 because node.prop is undefined.

Reply:
thank you @iain-s-h !